kubectl-rook-ceph.sh

```diff
@@ -35,6 +35,7 @@ function print_usage() {
   echo "COMMANDS"
   echo "  ceph <args>                               : call a 'ceph' CLI command with arbitrary args"
   echo "  rbd <args>                                : call a 'rbd' CLI command with arbitrary args"
+  echo "  health                                    : print cluster health status"
```

Comment:
IMO, a more descriptive help message would be useful.

For example: check health of the cluster and common configuration issues